src/resources/packages/wizard/components/tabs/display/tab.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { __ } from '@wordpress/i18n';
+import NextButton from '../../buttons/next';
+import SkipButton from '../../buttons/skip';
+import ViewCheckbox from './inputs/view-checkbox';
+
+const DisplayContent = ({closeModal, moveToNextTab, skipToNextTab}) => {
+	const views = [
+		'Month',
+		'Day',
+		'List',
+		'Week',
+		'Map',
+		'Photo',
+		'Summary',
+		'all'
+	];
+
+	document.addEventListener(
+		'change',
+		function(e) {
```

Comment:
```suggestion
	(event) => {
```

would also change from `e.` to `event.` in here.